api/build.gradle

```
@@ -1,3 +1,3 @@
 dependencies {
-    implementation 'com.google.code.gson:gson:2.8.0'
+    implementation 'com.google.code.gson:gson:2.8.7'
```

```suggestion
    api 'com.google.code.gson:gson:2.8.7'
```
The dependency is transitive after all